### PR TITLE
Fixed the bug within test data referenced by #185, also updated node …

### LINKED ISSUE #182: Add datestamp to activities table in db
☝️ Just incase

### LINKED ISSUE #185: 🐛warning when building local db
screen shot 2018-02-21 at 11 03 34

### PR DESCRIPTION
**Pull request by RogeredBacon**
Fixed the bug within DB test data referenced by #185, also updated node modules.
Also added date to activities table as referenced by #182.

fixes #185
fixes #182

### CODE REVIEW

File: react-backend/database/dbBuild.sql [hunk 1]
```diff
@@ -45,7 +45,8 @@ CREATE TABLE activities
   thursday BOOLEAN DEFAULT false,
   friday BOOLEAN DEFAULT false,
   saturday BOOLEAN DEFAULT false,
-  sunday BOOLEAN DEFAULT false
+  sunday BOOLEAN DEFAULT false,
+  date TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP
```
Comment (eliascodes) [inline react-backend/database/dbBuild.sql]: `date`/`DATE` is a key word in sql, use another column name

File: package.json [hunk 2]
```diff
@@ -27,18 +26,18 @@
...
     "eslint-config-prettier": "^2.9.0",
     "eslint-plugin-import": "^2.8.0",
     "eslint-plugin-jsx-a11y": "^5.1.1",
-    "eslint-plugin-react": "^7.5.1",
+    "eslint-plugin-react": "^7.7.0",
     "istanbul": "^0.4.5",
-    "nodemon": "^1.14.11",
-    "nyc": "^11.4.1",
+    "nodemon": "^1.15.1",
+    "nyc": "^11.5.0",
```
Comment (eliascodes) [inline package.json]: why do we have both `istanbul` _and_ `nyc`?

**Comment by eliascodes** [comment]
also using "fixes" or "closes" just before an issue reference automatically closes it once the PR is merged :wink:

File: react-backend/database/dbBuild.sql (hunk 1 above)
Comment (RogeredBacon) [reply react-backend/database/dbBuild.sql]: Coming from our talk I've opened an issue about this - #204 .

File: package.json (hunk 2 above)
Comment (eliascodes) [reply package.json]: ignore this, I've removed istanbul in #202

Comment (RogeredBacon) [reply package.json]: Sweet I was about to ask

**Comment by RogeredBacon** [comment]
> also using "fixes" or "closes" just before an issue reference automatically closes it once the PR is merged

Awesome I'll try it out

**Comment by astroash** [comment]
☝️ You can put it in the PR description or commit messasges. I prefer PR descriptions cos then you can taksie backsie if you chance your mind 😊

**Pull request merged**